src/Pages/Signup/Signup.jsx

```diff
@@ -0,0 +1,58 @@
+import { Navbar } from "../../Components/Navbar/Navbar";
+import { Link } from "react-router-dom";
+import "../Login/login.css";
+const Signup = () => {
+  return (
+    <div>
+      <Navbar />
+      <div class="login-wrapper">
+        <div class="login">
+          <h1>SignUp</h1>
+          <div class="login-credentials">
+            <div class="user-name">
+              <div class="first-name">
+                <p for="">First Name</p>
+                <input
+                  type="text"
+                  class="email-validate"
+                  placeholder="First Name"
+                />
+              </div>
+              <div class="last-name">
+                <p for="">Last name</p>
+                <input
+                  type="text"
+                  class="email-validate"
+                  placeholder="Last name"
+                />
+              </div>
+            </div>
+            <div class="email">
+              <p for="">Email Address</p>
+              <input
+                type="email"
+                class="email-validate"
```

Comment:
You should create react components for the code that is repeating and try to follow the DRY principle (Don't Repeat Yourself).